code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -23,12 +23,10 @@
 
 /obj/item/gun/ballistic/bow/proc/drop_arrow()
 	drawn = FALSE
-	if(!chambered)
+	if(chambered)
+		chambered.forceMove(drop_location())
 		chambered = magazine.get_round(keep = FALSE)
-		return
-	if(!chambered)
-		return
-	chambered.forceMove(drop_location())
+		chambered = null
```

Comment:
why are you setting chambered to null immediately after setting it to magazine.get_round(keep = FALSE)?

Reply:
Ah, I forgot to remove `chambered =`. It just needs the `magazine.get_round(keep = FALSE)` call - thank you for pointing this out. I can update this quickly, should I just commit & push or should I squash the commits? New to public repos :)